pkg/constants/constants.go

```diff
@@ -19,4 +19,13 @@ const (
 	StatusTriggered    = "triggered"
 	StatusAcknowledged = "acknowledged"
 	StatusResolved     = "resolved"
+
+
+	//ScheduleIDS for fetching oncalls as per pagerduty documentation (https://redhat.pagerduty.com/escalation_policies#PA4586M)
```

Comment:
@MitaliBhalla could you replace https://redhat.pagerduty.com/escalation_policies#PA4586M with `https://<host>.pagerduty.com/escalation_policies`

Reply:
ACK